frontend/app/src/components/history/transactions/TransactionContent.vue

```diff
@@ -236,7 +236,7 @@ const saveData = async (
   if ('identifier' in event) {
     return await editTransactionEvent(omit(event, 'evmChain'));
   }
-  return await addTransactionEvent(event);
+  return await addTransactionEvent(omit(event, 'evmChain'));
```

Comment:
the backend changes for this are in develop but the frontend are only in bugfixes.
😛 hopefully it won't create a merge conflict

Reply:
I guess I'll just cherry pick ur commit about this to develop. There are some error in develop without this commit 😅